l10n_it_withholding_tax/models/account.py

```diff
@@ -315,6 +315,12 @@ class AccountFiscalPosition(models.Model):
 class AccountInvoice(models.Model):
     _inherit = "account.invoice"
 
+    print_withholding_tax_in_pdf = fields.Boolean(
```

Comment:
Stavo riguardando le modifiche e ho pensato: non sarebbe meglio togliere questo campo e stampare i dati della ritenuta quando la ritenuta è presente in fattura e non farlo quando non lo è?
Dovrebbe essere sufficiente basarsi sul campo `withholding_tax` della fattura

Reply:
.. come ho un attimo me lo vedo

Reply:
Concordo con @elbati. Se si volesse mantenere il campo il default dovrà essere **False**

Reply:
@eLBati ...corretto ..ma mi sa che ho fatto un pò di casino con i commit 😬

Reply:
Sì, c'è un commit di troppo